gridstatus/ercot.py

```diff
@@ -983,13 +983,13 @@ def _get_todays_outlook_non_forecast(
         supply_demand_json = self._get_supply_demand_json()
         data = pd.DataFrame(supply_demand_json["data"])
 
-        # need to use apply since there can be mixed
-        # fixed offsets during dst transition
-        # that result in object dtypes in pandas
-        data["Interval End"] = data["timestamp"].apply(
-            lambda x: pd.to_datetime(x).tz_convert("UTC"),
-        )
-        data["Interval End"] = data["Interval End"].dt.tz_convert(self.default_timezone)
+        # Parse in UTC to then convert to local to avoid DST transition issues because
+        # of mixed timezones
+        data["Interval End"] = pd.to_datetime(
+            data["epoch"],
+            units="ms",
+            utc=True,
+        ).dt.tz_convert(self.default_timezone)
```

Comment:
Note: this isn't changing behavior since `data["timestamp"]` and `data["epoch"]` are equivalent.

Test with

```python
import requests
import pandas as pd

json = requests.get(
    "https://www.ercot.com/api/1/services/read/dashboards/supply-demand.json"
).json()

data = json["data"]

df = pd.DataFrame(data)

df["timestamp"] = df["timestamp"].apply(lambda x: pd.to_datetime(x, utc=True))
df["epoch"] = pd.to_datetime(df["epoch"], unit="ms", utc=True)

(df["timestamp"] == df["epoch"]).all()
```